openvm/tests/apc_builder_outputs/guest_top_block.txt

```diff
@@ -59,7 +59,7 @@ mult=is_valid * 1, args=[15360 * read_data_aux__base__prev_timestamp_1 + 15360 *
 mult=is_valid * 1, args=[write_base_aux__timestamp_lt_aux__lower_decomp__0_1, 17]
 mult=is_valid * 1, args=[15360 * write_base_aux__prev_timestamp_1 + 15360 * write_base_aux__timestamp_lt_aux__lower_decomp__0_1 - (15360 * from_state__timestamp_0 + 61440), 12]
 
-// Bus 6 (OPENVM_BITWISE_LOOKUP):
+// Bus 6 (BITWISE_LOOKUP):
```

Comment:
Should these somehow print as client specific?

Reply:
I think it's fine. Even for the once that are not specific, you need to know how exactly the client implements e.g. a memory bus interaction. So the entire file is pretty specific to the client.